src/utils/localizationBundle/index.js

```diff
@@ -0,0 +1,199 @@
+    accepted: {
+      action: 'Accept',
+      description: `This lesson has been accepted - it can be claimed.`,
+    },
+    requested: {
+      action: 'Request',
+      description: `This lesson has been requested - it can be claimed.`,
+    },
+    claimed: {
+      action: 'Claim',
+      description: `This lesson is claimed. Now it needs a video to be added and lesson details filled out.`,
+    },
+    submitted: {
+      action: 'Submit',
+      description: 'Sweet! This lesson has been submitted and is waiting for review to be approved.',
+    },
+    rejected: {
+      action: 'Ask for Changes',
+      description: 'Changes have been requested for this lesson. Please update the lesson as requested.',
+      title: 'Changes Needed',
+    },
+    updated: {
+      action: 'Apply Update',
+      description:  'Lesson updated! Waiting for review to be approved.',
+    },
+    approved: {
+      action: 'Approve',
+      description: 'This lesson has been approved.',
+    },
+    published: {
+      action: 'Publish',
+      description: `This lesson has been published. It is available publicly for students to view on egghead.io.`,
+    },
+    flagged: {
+      action: 'Flag',
+      description: `This lesson has been flagged - it needs to be revised or it will need to be retired.`,
+    },
+    revised: {
+      action: 'Revise',
+      description: `This lesson has revised and is no longer flagged.`,
+    },
+    retired: {
+      action: 'Retire',
+      description: `This lesson has been retired.`,
+    },
+  },
+
+  lessonGroups: {
+    instructorTitle: `%s's Lessons`,
+    allTitle: 'All Lessons',
+    fallback: 'No lessons to show',
+    action: 'Create a new lesson',
+    inProgress: {
+      title: 'In Progress',
+    },
+    inReview: {
+      title: 'In Review',
+      description: 'These lessons are waiting for review to proceed.',
+      selfApproval: `Since you have ${selfReviewThreshold}+ lessons published, you can review your own lessons.`,
+    },
+    inQueue: {
+      title: 'In Queue',
+      description: 'These lessons are in the publishing queue. The queue automatically publishes them from top to bottom. Lessons that are in a course are not shown here because they are held back until the entire course is published.',
+    },
+    published: {
+      title: 'Published',
+      description: 'Published lessons and courses are available publicly for students to view on egghead.io.',
+      action: 'View published content',
+    },
+  },
+
+  lesson: {
+    video: {
+      title: 'Video',
+      fallback: 'There is no video to show - waiting for one to be uploaded.',
+    },
+    state: 'State',
+    actions: 'Actions',
+    instructor: 'Instructor',
+    technology: 'Technology',
+    summary: 'Summary',
+  },
+
+  instructorGroups: {
+    title: 'Instructors',
+    fallback: 'No instructors to show',
+    unpublished: {
+      title: 'Unpublished',
+    },
+    published: {
+      title: 'Published',
+    },
+  },
+
+}
```

Comment:
I tried to organize this into useful chunks; some of it is reused if it is generic, but most of this is specific to the top key it is in; I followed the same "promotion" philosophy as we do with components and screens.